src/components/list.jsx

```diff
@@ -0,0 +1,31 @@
+import React, {Component} from 'react'
+import Item from './item.jsx'
+
+class List extends Component{
```

Comment:
Shouldn't this have props for the name and list items for generalizing?

Reply:
Not quite following this, we should review it together.